ci/deploy_app.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+IMAGE_TAG="${IMAGE_TAG:=latest}"
+REGION="${REGION:=europe-west2}"
+MIN_INSTANCES="${MIN_INSTANCES:=1}"
+MAX_INSTANCES="${MAX_INSTANCES:=1}"
+
+gcloud beta run deploy eq-questionnaire-launcher \
+    --project="${PROJECT_ID}" --region="${REGION}" --concurrency=250 --min-instances="${MIN_INSTANCES}" --max-instances="${MAX_INSTANCES}" \
+    --port=8000 --cpu=1 --memory=256M \
```

Comment:
Can we make concurrency, CPU and memory also configurable but set them with defaults?

Reply:
Made them configurable 👍.